metis-common/src/main/java/eu/europeana/metis/mongo/MongoClientProvider.java

```diff
@@ -78,36 +79,30 @@ public MongoClientProvider(String connectionUri, Function<String, E> exceptionCr
    *
    * @param properties The properties of the Mongo connection. Note that if the passed properties
    * object is changed after calling this method, those changes will not be reflected when calling
-   * @param clientSettingsBuilder The settings to be applied. The default settings will not be used. The
-   * caller can incorporate the default settings by using an client settings builder obtained from {@link
-   * #getDefaultClientSettingsBuilder()}. {@link #createMongoClient()}.
+   * @param clientSettingsBuilder The settings to be applied. The default settings will not be used.
+   * The caller can incorporate the default settings by using an client settings builder obtained
+   * from {@link #getDefaultClientSettingsBuilder()}. {@link #createMongoClient()}.
    */
-  public MongoClientProvider(MongoProperties<E> properties, Builder clientSettingsBuilder) {
+  public MongoClientProvider(MongoProperties<E> properties, Builder clientSettingsBuilder)
+      throws E {
```

Comment:
Shouldn't the exception be reflected in the javadoc?

Reply:
OK